src/main/java/io/paradoxical/cassandra/loader/DbScriptsRunner.java

```diff
@@ -43,7 +43,8 @@ public void run(Session rootSession) throws Exception {
         if (dbRunnerConfig.getCreateKeyspace() != null && dbRunnerConfig.getCreateKeyspace()) {
             try {
                 rootSession.execute("CREATE KEYSPACE " + dbRunnerConfig.getKeyspace() + "\n" +
-                                    "WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };");
+                                    "WITH REPLICATION = { 'class' : '" + dbRunnerConfig.getReplicationStrategy() +
+                                    "', 'replication_factor' : " + dbRunnerConfig.getReplicationFactor() +") };");
```

Comment:
So maybe the command line option should be the JSON blob for the entire replication section? That way any replication strategy and configuration can be handled.